controllers/front/callbackform.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Altapay module for Prestashop
+ *
+ * Copyright © 2020 Altapay. All rights reserved.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+require_once(_PS_MODULE_DIR_.'/altapay/lib/altapay/altapay-php-sdk/lib/AltapayCallbackHandler.class.php');
+require_once(_PS_MODULE_DIR_.'/altapay/helpers.php');
+
+class AltapayCallbackformModuleFrontController extends ModuleFrontController
+{
+    /**
+     * Method to add external assets
+     */
+    public function setMedia()
+    {
+        parent::setMedia();
+        $this->addCSS(($this->module->getPathUri()).'css/altapay.css', 'all');
+        $this->addCSS(($this->module->getPathUri()).'css/custom_css.css', 'all');
```

Comment:
Useless parenthecies.
```suggestion
        $this->addCSS($this->module->getPathUri().'css/altapay.css', 'all');
        $this->addCSS($this->module->getPathUri().'css/custom_css.css', 'all');
```